resources/zib-Patient.xml

```diff
@@ -0,0 +1,476 @@
+      </mapping>
+    </element>
+    <element id="Patient.name">
+      <path value="Patient.name" />
+      <short value="NameInformation" />
+      <definition value="Patient's full name.&#xD;&#xA;The cardinality differs from the zib to allow including several name elements with a different name.use each." />
+      <comment value="The cardinality mismatch between the zib (0..1) and FHIR (0..*) is explained by the presence of the HumanName.use element. A full name can be broken into parts using multiple instances of HumanName that have its own value for use." />
+      <alias value="Naamgegevens" />
+      <type>
+        <code value="HumanName" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-NameInformation" />
+      </type>
+      <mapping>
+        <identity value="zib-patient-v3.2-2020EN" />
+        <map value="NL-CM:0.1.6" />
+        <comment value="NameInformation" />
+      </mapping>
+    </element>
+    <element id="Patient.telecom">
+      <path value="Patient.telecom" />
+      <slicing>
+        <discriminator>
+          <type value="profile" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <short value="ContactInformation" />
+      <definition value="Patient’s telephone number(s) or e-mail address(es)." />
+      <comment value="The cardinality mismatch between the zib (0..1) and FHIR (0..*) is explained by the containers inside the ContactInformation model which are 0..*. FHIR ContactPoint does not have these containers." />
+      <alias value="Contactgegevens" />
+      <mapping>
+        <identity value="zib-patient-v3.2-2020EN" />
+        <map value="NL-CM:0.1.5" />
+        <comment value="ContactInformation" />
+      </mapping>
+      <mapping>
+        <identity value="zib-contactinformation-v1.2-2020EN" />
+        <map value="NL-CM:20.6.1" />
+        <comment value="ContactInformation" />
+      </mapping>
+    </element>
+    <element id="Patient.telecom:telephoneNumbers">
+      <path value="Patient.telecom" />
+      <sliceName value="telephoneNumbers" />
+      <type>
+        <code value="ContactPoint" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-ContactInformation-TelephoneNumbers" />
+      </type>
+    </element>
+    <element id="Patient.telecom:e-mailAddresses">
+      <path value="Patient.telecom" />
+      <sliceName value="e-mailAddresses" />
+      <type>
+        <code value="ContactPoint" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-ContactInformation-E-mailAddresses" />
+      </type>
+    </element>
+    <element id="Patient.gender">
+      <path value="Patient.gender" />
+      <short value="Gender" />
+      <definition value="Patient’s administrative gender. The CodeSpecification extension is added conform the profiling guidelines, because the zib codes cannot be exactly mapped to the required FHIR codes." />

```

Comment:
Volgens mij geven we deze informatie aan via description van de binding plus de extensie. Dat zou afdoende moeten zijn, toch? Ik zie de informatie overigens niet op Simplifier terugkomen gek genoeg.